src/modules/app.js

```diff
@@ -0,0 +1,65 @@
+/* eslint no-undefined: "error" */
```

Comment:
- [ ] OPTIONAL
You have **disabled ESLint checks** on parts of your code. Please enable all ESLint checks on your code and try modifying that portion of your code to adhere to ESLint standards. This will ensure a robust and clean code.